classes/Segmentation.php

```diff
@@ -48,7 +48,7 @@ public function __construct()
 		$this->tab = 'administration';
 		$this->version = '2.8';
 		$this->module_key = '986fb62d4efe6fb00788ecaefce96a1f';
-		$this->local_path = _PS_MODULE_DIR_.'mailjet';
+		$this->_path = _PS_MODULE_DIR_.'mailjet';
 
         $this->_addIncludes();
```

Comment:
This change will be part of our next release, could you please accept the current PR without it?